keymaster/ta/mbedtls_proxy.c

```diff
@@ -220,6 +220,8 @@ static keymaster_error_t convert_epoch_to_date_str(uint32_t sec,
 	days_since_epoch = epoch;
 	/* Calculating WeekDay */
 	ntp_week_day = week_days[days_since_epoch % 7];
+	/* workaround 'set but not used' warning */
+	ntp_week_day = ntp_week_day;
```

Comment:
If this variable is not being used anywhere, can we remove it all-together ?

Reply:
We can but just thought we might keep it in case it's needed later. Is it ok to add
`DMSG("ntp_week_day = %u", ntp_week_day);`
instead? Or do you prefer just removing it altogether?